scripts/index.js

```diff
@@ -21,62 +21,72 @@ groups.forEach(group => {
         this.weight = weight,
-        this.place = place
+        this.place = place,
+        this.src = src,
+        this.group = group
+        this.groupLink = groupLink
 }
 
 let mammalEchidna = new Animals("Mammals", "Echidna", 50, ["insects such as ants and termites", "beetle larvae", "worms"],
     "Echidnas, also called spiny anteaters, are walking contradictions. They are mammals, but they lay eggs. They are often classified as long- or sort-beaked, but don't have beaks at all, in the traditional sense; they have fleshy noses that can be either on the long side or rather short. They don't really look like true anteaters (Myrmecophaga tridactyla), either, and they are not closely related to them. They are spiny, though; their bodies are covered with hollow, barbless quills. Echidnas are monotremes, egg-laying mammals. The only other living monotreme is the platypus.",
-    76, 10, "Throughout Austalia");
+    76, 10, "Throughout Austalia", "./images/Echidna.jpg", 'Mammals', './html/Mammals.html');
 
 
 let mammalTasmanianDevil  = new Animals("Mammals", "Tasmanian Devil", 5, ["A predator", "eat meat from other animals such as wallabies and wombats"],
     "The Tasmanian devil (Sarcophilus harrisii) (palawa kani: purinina)[3] is a carnivorous marsupial of the family Dasyuridae. Until recently, it was only found on the island state of Tasmania, but it has been reintroduced to New South Wales in mainland Australia, with a small breeding population. The size of a small dog, the Tasmanian devil became the largest carnivorous marsupial in the world, following the extinction of the thylacine in 1936. It is related to quolls, and distantly related to the thylacine. It is characterised by its stocky and muscular build, black fur, pungent odour, extremely loud and disturbing screech, keen sense of smell, and ferocity when feeding. The Tasmanian devil's large head and neck allow it to generate among the strongest bites per unit body mass of any extant predatory land mammal. It hunts prey and scavenges on carrion.",
-    70, 10, "Tasmania");
+    70, 10, "Tasmania",  "./images/Tasmanian Devil.jpg", 'Mammals', './html/Mammals.html');
 
 
 let mammalquokka  = new Animals("Mammals", "Quokka", 10, ["Plant eaters", " shrubs", "grasses"],
     "he Quokka, Setonix brachyurus, was described by early Dutch explorer, Willem de Vlamingh, 'as a kind of rat as big as a common cat'. His first sighting of the Quokka was on an island off the mouth of the Swan River. He named the island Rottenest ('rat nest') in honour of this sighting. The island is now known as Rottnest Island. Essentially the Quokka looks very much like other wallabies.It has short, very coarse and thick grey-brown fur over most of the body with lighter parts underneath. Its facial features consist of a naked nose on a short, broad face with rounded furry ears.The tail is relatively short and mostly devoid of hair. In contrast, the hair on the feet extends to cover its claws.",
```

Comment:
Letter T is missing at beginning of the sentence

Reply:
Fixed. Thanks